libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Recognizers/EntityRecognizers/Microsoft.UrlEntityRecognizer.schema

```diff
@@ -1,7 +1,7 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botframework-sdk/master/schemas/component/component.schema",
     "$role": "implements(Microsoft.IEntityRecognizer)",
-    "title": "Confirmation Url Recognizer",
+    "title": "Confirmation Url recognizer",
```

Comment:
>Url [](start = 27, length = 3)

Techinically shouldn't this be URL since it is an abbrev?

Reply:
Fixed.